jakartaee-web-api/pom.xml

```diff
@@ -310,6 +238,19 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- Included only for Javadoc resolution (optional, non-transitive) -->
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <version>${jakarta.xml.bind-api.version}</version>
+            <optional>true</optional>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
 
```

Comment:
I think, this should become a required part in the Core (or Web Profile) in a future release (Jakarta EE 11) because XML is still in heavy use and is also required for some configurations (like Persistence and optionally in Servlet) - but that's a separate issue.

Good to fix it here now! But is the exclusion required - I don't think so. But I can verify this with a jQA run, to be sure.

Reply:
Jakarta XML Binding got dependency to Jakarta Activation 2.1.0 - this need to be maintained. So it looks like we need this exclusion until we have that solved first unfortunately.